.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Thank your for contributing to Theiagen's Public Health Bioinformatics repository!
+Please fill in the appropriate checklist below and delete whatever is not relevant.
+
+Documentation on how to contribute can be found at https://github.com/theiagen/public_health_bioinformatics#contributing-to-the-phb-workflows
+
+Please replace all '[ ]' with '[X]' to demonstrate completion.
+
+Please delete all text within '<>'. 
+-->
+Closes <Issue number>
+
+## :hammer_and_wrench: Changes Being Made
+
+<!--Here give examples of the changes you've made to this pull request. Include an itemized list if you can. It'll help the reviewer-->
+
+## :brain: Context and Rationale
+
+<!--What's the context of the changes? Where there any trade-offs you had to consider?-->
+
+## :ladder: Workflow/Task Steps
```

Comment:
the `:ladder:` emoji doesn't show up as a ladder 😢 
I'd recommend changing it to a clipboard: 📋 `:clipboard:`